src/mirage/index.ts

```diff
@@ -330,14 +330,14 @@ export const startMirage = ({ environment = 'development' } = {}) => {
               name: 'VMOperation Peak Duration',
               topic: 'vm_operations',
               description:
-                'Summary:\nNo excessively long VM operations were found in this recording (the longest was 23.774 ms).',
+                'Summary:\nNo excessively long VM operations were found in this Recording (the longest was 23.774 ms).',
```

Comment:
None of the strings within this object should be updated - these are all samples of actual automated analysis text copied from an actual run, and these texts are generated by the JMC rules analysis engine where we don't control which things are capitalized.